### DISCUSSION TITLE
The public directory (and docker)

### DISCUSSION THREAD

**Discussion by timlegge** [kimai/kimai#2276] [Questions & Answers]
Hi

I am using Kimai2 under Docker with the fpm image which requires /opt/kimai/public to be mounted in a volume so during an upgrade I need to refresh it.

My question is, does a kimai2 version ever update that directory during its life time or is it static by version?  That is as long as I create a new volume with the 1.12 version there is nothing that I need from the 1.8 version of /opt/kimai/public 

Tim

**Comment by kevinpapst** [comment]
Avatar images are generated in that directory, this is the only moving part in there.
They will be regenerated if missing => no problem to create that mount fresh during an update.

**Comment by timlegge** [comment]
Thanks.
